mobile/library/common/config/config.cc

```diff
@@ -538,5 +540,41 @@ stats_sinks: *stats_sinks
 R"(
         overload:
           global_downstream_max_connections: 0xffffffff # uint32 max
+#{custom_layers}
 )";
+
+const char* rtds_layer_insert = R"(
+    - name: {}
+      rtds_layer:
+        name: {}
+        rtds_config:
+          initial_fetch_timeout:
+            seconds: 1
+          resource_api_version: V3
+          api_config_source:
```

Comment:
it just occurred to me that for Traffic Director, it only supports ADS.  So we will also want to support ads instead of api_config_source: https://github.com/envoyproxy/envoy/blob/main/api/envoy/config/core/v3/config_source.proto#L225.  It would require setting the bootstrap's ads_config field: https://github.com/envoyproxy/envoy/blob/main/api/envoy/config/bootstrap/v3/bootstrap.proto#L101

I think we can start with only supporting ADS, and add non-ADS support later.  This will just require moving a bunch of this config stuff to `bootstrap.dynamic_resources.ads_config` and have `rtds_config` just set the `ads` field.